src/Mantasflowers.Services/Services/Email/EmailService.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Mantasflowers.Services.Services.Exceptions;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+namespace Mantasflowers.Services.Services.Email
+{
+    public class EmailService : IEmailService
+    {
+        private readonly ISendGridClient _sendGridClient;
+        private readonly SendgridConfiguration _sendGridConfig;
+
+        private const string EmailTemplatePath
+            = @"StaticData/EmailTemplate.html";
+
+        public EmailService(ISendGridClient sendGridClient, SendgridConfiguration sendGridConfig)
+        {
+            _sendGridClient = sendGridClient;
+            _sendGridConfig = sendGridConfig;
+        }
+
+        public async Task SendEmailAsync(
+            DateTime purchaseDate,
+            string clientFullName,
+            string clientEmail,
+            string orderNumber,
+            string orderUrl)
```

Comment:
Why not make this part into an easily mappable object?

Reply:
I had that idea. Not sure tdh, if it will only get called at one place (and it doesnt need to return anything), maybe its not worth it.

Reply:
That would be an inconvenience for a caller. Currently, code most likely wouldn't last long without being prone to change.